apps/extension/src/routes/popup/home/inspect-address/types.ts

```diff
@@ -0,0 +1,3 @@
+export type Result =
```

Comment:
Can we use a more specific name than Result here?

Reply:
Yep, renamed to `AddressOwnershipInfo`